environ.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"os"
+	"strings"
+)
+
+const clientsKey = "XLANG_CLIENTS"
+const paramKey = "XLANG_PARAMS_"
+
+func ReadMatrixFromEnviron() Matrix {
+
+	clients := strings.Split(os.Getenv(clientsKey), ",")
+
+	params := make(Params, 0)
+	paramKeys := make(ParamKeys, 0)
+
+	for _, e := range os.Environ() {
+
+		found := strings.Index(e, paramKey)
+		if found == -1 {
+			continue
+		}
+		d := strings.Replace(e, paramKey, "", -1)
```

Comment:
Don't replace all occurrences. Just the first.

https://golang.org/pkg/strings/#TrimPrefix